src/extension/background-script/CryptoServices/utils.ts

```diff
@@ -28,20 +31,43 @@ export interface TypedMessageText extends TypedMessageMetadata {
     readonly type: 'text'
     readonly content: string
 }
+export interface TypedMessageImage extends TypedMessageMetadata {
+    readonly type: 'image'
+    readonly content: string
```

Comment:
Eh, why the content of an image is a string?

Reply:
Cause we need images to be transferred between background and content page.

Reply:
We can transfer `ArrayBuffer` or `Uint8Array`.

Reply:
I will take it into my backlog. I can't remember why I choose `string`. Maybe there were some issues in FF.

Reply:
I recalled something. It seems like there is a problem for transfering TypedArray in Firefox